tests/GeoNames/ClientTest.php

```diff
@@ -46,6 +46,66 @@ public function testGetSupportedEndpoints()
         $this->assertContains('wikipediaSearch', $endpoints);
     }
 
+    public function testGetLastTotalResultsCountAssertNull()
```

Comment:
What is the test case here? e.g. `testGetLastTotalResultsWhenNoResults`

Reply:
Just making sure, that the value is initialised correctly. It was maybe more sensible to do when all tests where in one single function to check if we start with the expected base point ...